google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -432,6 +432,7 @@ def delete force: nil
         #   length is 1,024 characters.
         # @param [String] name A descriptive name for the table.
         # @param [String] description A user-friendly description of the table.
+        # @param [EncryptionConfiguration] encryption_configuration A configuration for custom table data encryption.
```

Comment:
@tswast This can be broken up like line 430:
# @param [EncryptionConfiguration] encryption_configuration A configuration for custom 
#   table data encryption.

Reply:
Thanks.

FYI: I hit the "edit" button on your comments since the H1 headers instead of code blocks were bugging me. :-)